Content.Server/GameObjects/Components/Weapon/Ranged/Barrels/ServerRangedBarrelComponent.cs

```diff
@@ -206,6 +209,29 @@ private void Fire(IEntity shooter, GridCoordinates target)
                 recoilComponent.Kick(-angle.ToVec() * 0.15f);
             }
 
+            //Explode in the face of the clown!! ;^)
+            if(shooter.HasComponent<ClumsyComponent>())
+            {
+
+                soundSystem.PlayAtCoords("/Audio/Items/bikehorn.ogg",
+                Owner.Transform.GridPosition, AudioParams.Default, 5);
+
+                soundSystem.PlayAtCoords("/Audio/Weapons/Guns/Gunshots/bang.ogg",
+                Owner.Transform.GridPosition, AudioParams.Default, 5);
+
+                var health = shooter.GetComponent<DamageableComponent>();
```

Comment:
Maybe put this behind a `TryGetComponent`?

Reply:
i did that at first actually.